forecasting/more_models.py

```diff
@@ -0,0 +1,339 @@
+import pandas as pd
+from prophet import Prophet
+from statsforecast.models import AutoETS, AutoARIMA
+from .normal_naive_models import naive_pi, drift_pi, mean_pi
+from .bootstrap_naive_models import forecast_dates, naive_error, drift_error, mean_error
+from sklearn.metrics import *
+from statsmodels.tsa.seasonal import MSTL
+
+
+
+def prophet_fit(df:pd.DataFrame, target_col:str,**kwargs) -> pd.DataFrame:
```

Comment:
Add Prophet() class. Remove prophet_fit(), prohet_forecast() and have these as seperate methods (.fit(), .predict()). Possibly read through paper to see if easier way to extract prediciton intervals.